Author: Ole Streicher <debian@liska.ath.cx>
Description: The configuration directory is /etc, independent of
 the prefix. This was set wrong in er_prefix.h and is corrected
 here.
--- a/src/er_prefix.h.in
+++ b/src/er_prefix.h.in
@@ -3,5 +3,5 @@
 MYDATA = @datadir@
 ESOREX_PREFIX=@prefix@
 */
-#define ESOREX_PREFIX "@prefix@/etc/"
+#define ESOREX_PREFIX "@sysconfdir@"
 
